Tasks/PublishBuildArtifacts/task.json

```diff
@@ -2,7 +2,7 @@
   "id": "2FF763A7-CE83-4E1F-BC89-0AE63477CEBE",
   "name": "PublishBuildArtifacts",
   "friendlyName": "Publish Build Artifacts",
-  "description": "Publish Build artifacts to the server or a file share",
+  "description": "Publish build artifacts to TFS/Visual Studio Team Services or a file share",
```

Comment:
Visual Studio Team Services/TFS - in many places we put VSTS first in the strings and it will be good to be consistent